.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "6"
   - "8"
   - "10"
```

Comment:
Could you add Node 11 and 12 which are both currently supported?

https://nodejs.org/en/about/releases/